tests/app/celery/v3/test_notification_tasks.py

```diff
@@ -48,7 +48,9 @@ def test_v3_process_notification_no_template(notify_db_session, mocker, sample_s
         assert notification_failure.body['phone_number'] is None
         assert notification_failure.body['email_address'] == 'test@va.gov'
     finally:
-        notify_db_session.session.delete(notification_failure)
+        # Teardown
+        stmt = delete(NotificationFailures).where(NotificationFailures.notification_id == request_data['id'])
+        notify_db_session.session.execute(stmt)
```

Comment:
do we need a tech debt ticket to make a global method for DB tear down - something that clears our db in between tests?

Reply:
In `tests/conftest.py`, `pytest_sessionfinish` is capable of truncating everything. If a test does not self-tear down we consider that a failing test. 

`pytest_sessionfinish` scans all tables looking for leftover rows. Artifacts left for subsequent tests led to a lot of problems in the past.